src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -71,7 +76,12 @@ export default function ColorPickerControl({
                 data-testid="bp-ColorPickerControl-palette"
                 {...handlers}
             >
-                <ColorPickerPalette colors={colors} data-testid="bp-ColorPickerPalette" onSelect={handleSelect} />
+                <ColorPickerPalette
+                    colors={colors}
+                    data-testid="bp-ColorPickerPalette"
```

Comment:
This isn't related to your change, but does this get applied to the palette? It's not a listed prop and we don't have a `...rest` operator anywhere. Can we remove it safely?